tests/functional/sngtestmod.py

```diff
@@ -39,6 +39,8 @@ def is_open(self):
 
     def send(self, msg):
         with open('test-python.log', 'a') as f:
+            if (isinstance(msg['MSG'], bytes)):
```

Comment:
Why do we need this check?

Reply:
I wanted to branch the `python2` from the `python3`, but yes, indeed it is not needed, as both of them use bytes... My bad.